### PR TITLE
Apester fix

### PR DESCRIPTION
**Pull request by chessky**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 
+import org.openqa.selenium.TimeoutException;
+
 public class Navigate extends BasePageObject {
 
   public Navigate toPage(String pageName) {
-    driver.get(urlBuilder.getUrlForWikiPage(pageName) + urlBuilder.getCacheBusterQuery(pageName));
+    try {
+      driver.get(urlBuilder.getUrlForWikiPage(pageName) + urlBuilder.getCacheBusterQuery(pageName));
+    }catch (TimeoutException e){
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Navigate.java#L28](https://github.com/Wikia/selenium-tests/blob/804c90bf8ad0532e34e96fffb7031e6564c5099f/src/test/java/com/wikia/webdriver/elements/common/Navigate.java#L28): Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+import org.openqa.selenium.TimeoutException;
+
 public class Navigate extends BasePageObject {
 
   public Navigate toPage(String pageName) {
-    driver.get(urlBuilder.getUrlForWikiPage(pageName) + urlBuilder.getCacheBusterQuery(pageName));
-
+    try {
+      driver.get(urlBuilder.getUrlForWikiPage(pageName) + urlBuilder.getCacheBusterQuery(pageName));
+    }catch (TimeoutException e){
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Navigate.java#L27](https://github.com/Wikia/selenium-tests/blob/bf08d2c589334cbdee72c155325221ced221cf21/src/test/java/com/wikia/webdriver/elements/common/Navigate.java#L27): Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Navigate.java#L26](https://github.com/Wikia/selenium-tests/blob/9649a7dc90daef65a930900e3d5ce6922e0c387f/src/test/java/com/wikia/webdriver/elements/common/Navigate.java#L26): Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Navigate.java#L28](https://github.com/Wikia/selenium-tests/blob/05a3373122342a6eaee005d4c8869c09dc65c7a9/src/test/java/com/wikia/webdriver/elements/common/Navigate.java#L28): Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 3]
```diff
@@ -19,6 +26,7 @@ public Navigate toPageByPath(String path) {
       Thread.sleep(2000);
     } catch (InterruptedException e) {
       e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![CRITICAL][CRITICAL] 1 critical

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 4]
```diff
@@ -1,12 +1,21 @@
 package com.wikia.webdriver.elements.common;
 
+import com.wikia.webdriver.common.core.elemnt.JavascriptActions;
+import com.wikia.webdriver.common.logging.Log;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import org.openqa.selenium.TimeoutException;
+
+import java.util.logging.Logger;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: MINOR Remove this unused import 'java.util.logging.Logger'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/elements/common/Navigate.java [hunk 5]
```diff
@@ -1,12 +1,19 @@
...
+import org.openqa.selenium.TimeoutException;
 
 public class Navigate extends BasePageObject {
 
   public Navigate toPage(String pageName) {
-    driver.get(urlBuilder.getUrlForWikiPage(pageName) + urlBuilder.getCacheBusterQuery(pageName));
-
+    try {
+      driver.get(urlBuilder.getUrlForWikiPage(pageName) + urlBuilder.getCacheBusterQuery(pageName));
+    }catch (TimeoutException e){
```
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/elements/common/Navigate.java]: I guess there's something wrong with your styling settings.

**Pull request merged**